src/rdkafka.c

```diff
@@ -2851,9 +2855,11 @@ void rd_kafka_rebootstrap(rd_kafka_t *rk) {
 }
 
 /**
- * Restarts rebootstrap timer with the configured interval.
+ * Restarts rebootstrap timer with the configured interval. Only if not
+ * started or stopped.
  *
  * @locks none
+ * @locks_acquired rd_kafka_timers_lock()
  * @locality any
  */
 void rd_kafka_rebootstrap_tmr_restart(rd_kafka_t *rk) {
```

Comment:
Better to name it `rd_kafka_rebootstrap_tmr_try_start` now as we don't ever restart it.

```suggestion
void rd_kafka_rebootstrap_tmr_try_start(rd_kafka_t *rk) {
```